packages/app/docs/stories/appCrashes.story.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable react/jsx-no-undef */
+import React from 'react';
+import RainbowFirebaseApp from '../../src/components/App';
+
+export const DefaultCrashes = () => {
+    return (
+        <RainbowFirebaseApp>
+            <Error />
+        </RainbowFirebaseApp>
+    );
+};
```

Comment:
here we should add an example that uses various ErrorBoundaries at different levels